pkg/bindings/containers/containers.go

```diff
@@ -24,7 +24,7 @@ var (
 // the most recent number of containers.  The pod and size booleans indicate that pod information and rootfs
 // size information should also be included.  Finally, the sync bool synchronizes the OCI runtime and
 // container state.
-func List(ctx context.Context, filters map[string][]string, all *bool, last *int, pod, size, sync *bool) ([]entities.ListContainer, error) { // nolint:typecheck
```

Comment:
I prefer to keep it as it would break existing users. I'd rather mark it as deprecated and remove it with 3.0.

Reply:
This is just in the bindings code; do we consider that public API? The actual API still handles the request without issue.

Reply:
i think we'd be bad players to break existing users. on the other hand, we do not want to mislead them.  how would you feel about leaving it but noting in the docs (above the func) that it is a no-op ?

Reply:
I really want the bindings to be stable. @lsm5 is writing a blog post about them as we wanted to promote the bindings to Go developers. The blog does not use containers.List but could have.